src/views/Clients/ViewClients/index.jsx

```diff
@@ -30,8 +32,11 @@ import clientsQuery from './clients.graphql';
   },
 }))
 export default class ViewWorker extends PureComponent {
+  componentDidMount() {
+    this.handleClientSearchSubmit();
+  }
   state = {
-    clientSearch: '',
+    clientSearch: this.props.user.credentials.clientId,
```

Comment:
If you log out and then refresh the page, the view will crash on this line. This is because `this.props.user` is undefined.

You can do:

```jsx
clientSearch: this.props.user ? this.props.user.credentials.clientId || '',
```